<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerWindow</class>
 <widget class="QMainWindow" name="ImageViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>openMittsu Image Viewer</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="PixmapLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>798</width>
      <height>557</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string notr="true">Image here</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_to_Clipboard"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionReset_Zoom"/>
    <addaction name="actionFit_to_Window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_as">
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCopy_to_Clipboard">
   <property name="text">
    <string>Copy to Clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionFit_to_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QLabel</extends>
   <header>widgets/PixmapLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
